recipes/default.rb

```diff
@@ -97,9 +108,23 @@
   )
 end
 
-service 'chef-guard' do
-  provider Chef::Provider::Service::Upstart
-  supports :restart => true, :reload => true, :status => true
-  action :start
-  only_if { File.exists?("#{node['chef-guard']['install_dir']}/chef-guard") }
+# Check if CentOS/RHEL 7 is used
+if node['platform_family'] == 'rhel' && node['platform_version'].to_i >= 7 && node['platform_version'].to_i < 8
```

Comment:
As this cookbook also supports Ubuntu and Ubuntu 15.04 and newer is also using `systemd` as the default init system, we should add this fix for Ubuntu > 15.04 as well...